src/osp/Active/SysWire.cpp

```diff
@@ -51,15 +51,15 @@ void WireInput::doErase()
      list()->cut(this);
 };
 
-SysWire::SysWire(ActiveScene &scene) :
-        m_updateWire(scene.get_update_order(), "wire", "", "",
-                     std::bind(&SysWire::update_propigate, this))
+SysWire::SysWire(ActiveScene &scene)
+ : m_updateWire(scene.get_update_order(), "wire", "", "",
+                [this] (ActiveScene& rScene) { this->update_propagate(rScene); })
```

Comment:
From a "don't repeat yourself" mindset, this same specific function gets called inside of a lambda in multiple places.

Makes me think that it would be better to have a specific struct that takes a pointer to the base class of SysWire and has an operator() function.

Reply:
With ActiveScene as an argument, many of these update functions will be turned into static functions soon, removing the need for this lambda in the first place.